src/reference/config/bindings/http/.partials/options.md

```diff
@@ -0,0 +1,135 @@
+
+#### options.requests
+
+> `array` of `object`
+
+<!-- todo: Dev input -->
```

Comment:
@jfallows I don't know what to say about the `requests` and `responses` arrays here. The individual parts are straight forward, but I don't know what they do in the `http` binding. Unless they are just for defining validation types.

Reply:
@vordimous yes, they define the validation types that should be enforced by the `http` binding, independent of routing.

For example, we populate these automatically from openapi or asyncpi (http).